conbench-config.yml

```diff
@@ -13,3 +13,5 @@ data:
   DB_PORT: "{{DB_PORT}}"
   FLASK_APP: "{{FLASK_APP}}"
   DISTRIBUTION_COMMITS: "{{DISTRIBUTION_COMMITS}}"
+  # Hardcoding this for now. We can always make it configurable in the future.
+  PROMETHEUS_MULTIPROC_DIR: /tmp/_conbench-promcl-coord-dir
```

Comment:
:rocket: Should work, yes! Analogous to https://github.com/conbench/conbench/blob/192b8b1ca8f2b39b36977e56683e34f4b5196a49/ci/minikube/conbench-config-for-minikube.yml#L24.

Once again thank you, I should have crunched through those hoops :)

Reply:
haha, no problem - I just set up the Buildkite pipeline to email me about `conbench-staging-deploy` builds. It's too often that I forget about that pipeline and failures show up!